benchmark_runner/benchmark_operator/benchmark_operator_workloads.py

```diff
@@ -793,7 +801,7 @@ def run_workload_func(self, workload_full_name: str):
         self.__template.generate_yamls(workload=workload_full_name)
         if 'hammerdb' in workload_full_name:
             # check if ocs is installed
-            if self.__environment_variables_dict.get('ocs_pvc', '') == 'True':
+            if self.__environment_variables_dict.get('ocs_pvc', True):
```

Comment:
See environment_variables.py:52.  I've already set that via `env_to_bool`; part of this project is so that none of the other code has to do string comparisons for booleans.

Reply:
Ok, but according to Python:
dictionary.get("bogus", default_value) -> the second value is always the default.
BTW, self._environment_variables_dict['endpoint_url'] = os.environ.get('IBM_ENDPOINT_URL', None)
MUST be None for unittest, need to find a way to solve it.